libr/bin/format/elf/elf.h

```diff
@@ -94,6 +94,7 @@ typedef struct Elf_(r_bin_elf_dynamic_info) {
 	Elf_(Xword) dt_flags_1;
 	Elf_(Xword) dt_rpath;
 	Elf_(Xword) dt_runpath;
+	RVector *dt_needed;
```

Comment:
actually we could avoid an allocation by using `RVector dt_needed` and using `r_vector_init`/`r_vector_clear` instead of `r_vector_new`/`r_vector_free`.